dds_registration/models.py

```diff
@@ -86,14 +86,29 @@ def __init__(self, *args, **kwargs):
 
 
 class Membership(models.Model):
-    user = models.ForeignKey(User, related_name='registrations', on_delete=models.CASCADE)
-    started = models.IntegerField(default=lambda: date.today().year)
-    until = models.IntegerField(default=lambda: date.today().year)
+    user = models.ForeignKey(User, related_name='memberships', on_delete=models.CASCADE)
```

Comment:
Please revert back to previous behaviour, it was chosen like that for a reason.

Reply:
Can you pls fix your code and provide its working version then? Probably, it'd be better to drop my fixes and to use your own updated revision.